src/main/target/ALIENFLIGHTF4/system_stm32f4xx.c

```diff
@@ -433,7 +433,7 @@ uint32_t hse_value = HSE_VALUE;
   */
 
 #if defined(STM32F40_41xxx)
-  uint32_t SystemCoreClock = 168000000;
+  uint32_t SystemCoreClock = 192000000;
```

Comment:
Yup, doesn't seem to be any problem. No extra heat is noted and flights are fine.The STM32F4's default clock seems quite conservative. It's providing some interesting results for MS1.25.